include/uv.h

```diff
@@ -1160,6 +1162,10 @@ struct uv_cpu_info_s {
   struct uv_cpu_times_s cpu_times;
 };
 
+struct uv_battery_info_s {
+  int charging;
```

Comment:
Ambiguous name, can be interpreted as a boolean. Consider naming it e.g. `charge_time` or `charge_time_in_secs`.

Also, since it can't go negative, it's probably better to make it an `unsigned int`. Maybe make it a `uint64_t` out of an abundance of caution.

You probably want to add `void *reserved[7];` so there's space to add more features, because, once it's released, the size of the struct can never, ever change again.

Reply:
> Because, once it's released, the size of the struct can never, ever change again.

@bnoordhuis Why can't the size of the struct can never change?

Reply:
ABI stability.